x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/eventing/windows.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EventingCheckbox } from './checkbox';
+import { OS, EventingFields } from '../../../../types';
+import { ConfigForm } from '../config_form';
+
+export const WindowsEventing = React.memo(() => {
+  const checkboxes = [
```

Comment:
maybe cache these? use `useMemo()`

Reply:
if its wrapped around a react.memo() like in line 13, do you still have to do another memo?

Reply:
Yes, but it's a different method. 
`React.memo` is like our old `PureComponent` - it generates React component from the render function you provide.

React's `useMemo()` will memoize the return value of the factory function you provide on input so that on each re-render of the component, if the dependencies of that factory function have not changed, then the memoized value is returned. (`useCallback()` works the same way).

Reply:
@parkiino Just a reminder, this still needs `useMemo`. Or move it out of the component definition entirely, since its not being calculated based on any state.

Reply:
ok